data/constants/keycodes/keycodes_0.0.1_basic.hjson

```diff
@@ -1510,6 +1510,14 @@
                 "KC_RCMD",
                 "KC_RWIN"
             ]
+        },
+        "0x00E8": {
+            "group": "media",
+            "key": "KC_AIRPLANE_MODE",
+            "label": "Toggle Airplane Mode",
+            "aliases": [
+                "KC_AIRP"
+            ]
         }
```

Comment:
Please move just the new keycode to a new file with 0.0.2 in the filename instead.
It'll get merged with all the others to create the final definition -- 0.0.1 is already "locked in".